embedded_localization.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
 
   spec.metadata["homepage_uri"] = spec.homepage
   spec.metadata["source_code_uri"] = spec.homepage
-  spec.metadata["changelog_uri"] = "https://github.com/tilo/embedded_localization/blob/main/CHANGELOG.md"
```

Comment:
I'm not sure why this code is part of my PR as I didn't made that change :thinking:

Reply:
i pushed that after the release of the last gem version.. 
You probably didn't pull the latest main